Libraries/LibJS/Runtime/DatePrototype.cpp

```diff
@@ -146,4 +147,14 @@ Value DatePrototype::get_time(Interpreter& interpreter)
     return Value(static_cast<double>(seconds * 1000 + milliseconds));
 }
 
+Value DatePrototype::to_string(Interpreter& interpreter)
+{
+    auto* this_object = this_date_from_interpreter(interpreter);
+    if (!this_object)
+        return {};
+    // FIXME: Deal with timezones once SerenityOS has a working tzset(3)
+    auto string = this_object->datetime().to_string("%a %b %d %Y %T GMT+0000 (UTC)");
+    return Value(interpreter.heap().allocate<PrimitiveString>(move(string)));
```

Comment:
Let's use the convenience helper `js_string` here:
```suggestion
    return js_string(interpreter.heap(), move(string));
```